doc/Usage.md

```diff
@@ -0,0 +1,26 @@
+# Using Custom Student Verification App
+
+## Submitting Identification Document
+
+Once installed and enabled, learners should see the following identification document upload form when trying to verify themselves -
```

Comment:
@shimulch, @aayushagra, to show the custom form, some theme changes to the template(s) are needed right? Can you add some details about that?

Reply:
Documentation regarding what needs to be changed in templates added.